modules/openy_features/openy_location/modules/openy_loc_branch/openy_loc_branch.install

```diff
@@ -180,3 +180,33 @@ function openy_loc_branch_update_8005() {
     'rabbit_hole.behavior_settings.node_type_branch',
   ]);
 }
+
+/**
+ * Update view display for facility page.
+ */
+function openy_loc_branch_update_8006() {
+  $config_dir = drupal_get_path('module', 'openy_loc_branch') . '/config/install/';
+  // Import new configuration
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
+    'core.entity_view_display.node.branch.facility',
+    'core.entity_view_mode.node.facility',
```

Comment:
This config exists in separate module, so this update will never update this configuration.